src/scripts/show_originals.js

```diff
@@ -6,36 +6,57 @@ import { keyToCss } from '../util/css_map.js';
 
 const excludeClass = 'xkit-show-originals-done';
 const hiddenClass = 'xkit-show-originals-hidden';
+const activeTimelineClass = 'xkit-show-originals-timeline';
 const lengthenedClass = 'xkit-show-originals-lengthened';
 
-const dashboardTimeline = '/v2/timeline/dashboard';
-const dashboardTimelineRegExp = new RegExp(dashboardTimeline);
-
-const lengthenTimelines = async () => {
-  const timeline =
-    document.querySelector(`[data-timeline="${dashboardTimeline}"]:not(.${excludeClass})`);
+let showOwnReblogs;
+let showReblogsWithContributedContent;
+let whitelistedUsernames;
+let runOnDashboard;
+let runOnPeepr;
+let runOnBlogSubscriptions;
 
-  if (timeline) {
-    timeline.classList.add(excludeClass);
-    const paginationCss = await keyToCss('manualPaginatorButtons');
+const lengthenTimeline = async (timeline) => {
+  const paginationCss = await keyToCss('manualPaginatorButtons');
```

Comment:
```suggestion
  const paginatorSelector = await keyToCss('manualPaginatorButtons');
```

this makes me realise how poorly-named `keyToCss` is. will have to do a repo-wide search and replace sometime

Reply:
Yeah, agreed. I was trying to think of something extremely short that would be sufficiently clear because of how it now gets used inline with resolveExpressions so much.

Reply:
Oh, that's a factor I hadn't considered. I was thinking `keyToSelector`, but that would be clunky written out multiple times in the same line.

Considering what `keyToCss` returns now, how about `keyToIs`? Not sure if that's too esoteric.

Reply:
Hmm... yeah, that feels a bit too esoteric imo, not that I have any great ideas. I would assume that didn't do the translation step, just converted an array of classes to an is selector.

`import { cssMap } from '../util/css_map.js';` - kinda cute; kinda weird to have to rename the storage variable inside the module
`import { obfuscate } from '../util/css_map.js';` - fairly obscure
`import { map } from '../util/css_map.js';` - nice if you only ever use it inline in a template string that is obviously going to be a stylesheet; very strange in other contexts

I guess there's always
```js
export const keyToSelector = /* */;
export const map = keyToSelector;
```

or making a pattern of using `import { keyToSelector as map } from '../util/css_map.js';` where relevant, but... I dunno